### PR TITLE
Started work on user profile page

### PR DESCRIPTION
**Pull request by chdefrene**
Added some bare-bones functionality to user profile page.
Also fixed some typo in the KSG position enum.

### CODE REVIEW

**Review by alexaor — APPROVED** [review]

File: users/models.py [hunk 1]
```diff
@@ -56,6 +56,20 @@ def __str__(self):
     def __repr__(self):
         return "User(name=%s)" % (self.first_name,)
 
+    def full_name(self):
```
Comment (trmdy) [inline users/models.py]: This method can probably be removed, as there is a built in method `get_full_name` which is more or less equivalent to this.

File: users/views.py [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 from __future__ import unicode_literals
 
-from django.shortcuts import render
+from django.views import generic
 
-# Create your views here.
+from users.models import User
+
+
+class CurrentUserView(generic.ListView):
```
Comment (trmdy) [inline users/views.py]: I would prefer to have this (and other non-API-views) as method-based views and not class-based views. If you disagree we could of course have a discussion about it.

**Review by trmdy — APPROVED** [review]
Apart from the two small comments it looks good.

File: users/models.py (hunk 1 above)
Comment (chdefrene) [reply users/models.py]: OK, will fix

File: users/views.py (hunk 2 above)
Comment (chdefrene) [reply users/views.py]: I agree. Class-based views has become a habit for me, but after looking at function-based views, they are much more suited for this project. For API views I would argue that class-based is better

**Pull request merged**